docs/updating_schemas.md

```diff
@@ -236,6 +324,75 @@ To add an object with accession prefix 'SM':
              }
 
 
+-----
+
+Schema property guidelines
+----------------
+
+* When naming a property, try to avoid overly long names.
+* Array properties must have a plural property name: samples, not sample.
+
+Below are the required and optional components divided according the property's `type`.
+* All properties:
+    * `title`: Required. A readable name of the property, in Title Case.
+    * `type`: Required. `string`, `number`, `integer`, `boolean`, `array`, or `object`.
+    * `description`: Required. A definition of the property.
+    * `default`: A default value automatically assigned if no value is provided upon submission.
+    * Strings:
+        * `enum`: A list of valid values when a property requires a controlled vocabulary.
+        * `linkTo`: A list of valid objects to link to. Must be the name of a schema, which can be abstract or concrete.
+        * `pattern`: A regex pattern applied to the string.
+    * Numbers or integers:
+        * `minimum`: A minimum value.
+        * `maximum`: A minimum value.
+    * Arrays:
+        * `items`: Required. A definition of the items that are valid in the array. Must include `type` at minimum.
+        * `minItems`: Required. An integer indicating the minimum number of items in an array. Must be at least 1.
+        * `maxItems`: An integer indicating the maximum number of items in an array.
+        * `uniqueItems`: Required. A boolean which is always `True`.
+    * Objects:
+        * `properties`: A nested schema. Must obey above requirements for each property: `title`, `type`, and `description`.
+
+The following are properties categorized by IGVF members' permission to submit. These properties must follow the preceding guidelines according to their `type` in addition to the guidelines below.
+* Submittable properties:
+    * `comment`: Required. Instructions for submitters explaining details of how to submit the property. If there is a dependency defined for the property, the comment must include: "This property is affected by dependencies. See \[link\]."
+    * `submissionExample`: Required. An example of how to specify the property using appscript and igvf_utils tools.
+* Admin only properties:
+    * `permission`: Required. A string property which is always `import_items`. Must not be present for any property that can be modified by any submitter.
```

Comment:
not required, optional

Reply:
I put required here, but the intent was it should only apply to "admin only" properties. Is there a better way to make that more clear?

Reply:
ohh i didn't see the indent. I think that's fine.